imports/client/TestRunner.js

```diff
@@ -0,0 +1,190 @@
+/* USAGE INSTRUCTION:
+ To call tests: simply import the TestRunner into your .js file and use the run method of this class
+ To add tests: have a look at the TestCase class. Simply add more tests by expanding the createTestCases-Method.
+ To add new scenarios triggering tests: simply add an unique identifier for your scenario to TestRunner.TRIGGERS, then create your tests.
+ */
+import {ConfirmationDialogFactory} from '../../client/helpers/confirmationDialogFactory';
+
+export class TestRunner {
```

Comment:
The name "TestRunner" in SW-Projects is normally claimed by a Unite/E2E TestRunner. As well as the name "TestCase". So, could you please rename these two classes to something not so confusing. Like: e.g, QualityTestRunner and QualityTestCase?